test/controllers/DeparturesControllerSpec.scala

```diff
@@ -272,6 +277,7 @@ class DeparturesControllerSpec extends SpecBase with ScalaCheckPropertyChecks wi
 
           val result = route(application, request).value
 
+          contentAsString(result) mustEqual "Failed to create Departure with the following error: InvalidRootNode(Node InvalidRootNode didn't match CC015B)"
```

Comment:
The error message generated could be changed to the following:<br>`The root element name does not match 'CC015B'`